inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -260,6 +260,8 @@ StatusCode CNNNetworkNGraphImpl::addOutput(const std::string& layerName, size_t
 void CNNNetworkNGraphImpl::addOutput(const ::ngraph::Output<::ngraph::Node> & output) {
     auto dataName = ngraph::op::util::create_ie_output_name(output);
     DataPtr data;
+    if (_data.count(dataName))
+        data = _data[dataName];
```

Comment:
nit: in general we can remove `if` and just call `data = _data[dataName];` because the call of `_data[dataName]` creates a new DataPtr in the map if it doesn't exist.
Moreover in this case we can remove line [266](https://github.com/openvinotoolkit/openvino/pull/1888/files#diff-9fde058e42f3c1a942f6c4482230ccd6R266).